doc/source/development/packaging.rst

```diff
@@ -0,0 +1,98 @@
+==================
+Packaging releases
+==================
+
+OpenGHG is now completely tested and deployed using GitHub actions.
+The most updated developer code lies in the `OpenGHG repository <https://github.com/openghg/openghg>`_
+
+The development process should be:
+
+* New changes to any section of the code base should always be made using feature branches.
+
+.. code-block:: bash
+
+   git checkout -b branch_name
+
+.. note::
+   branch_name - Iss(Issue number) recommended if the change is a bug fix for an issue. E.g. “Iss123_add_attributes”. May also want to include “fix_” and “feature_” as appropriate.
```

Comment:
I don't think we need this as a note. Maybe an example below with a `git checkout -b Iss961/updateReleaseDocs` or something.